src/metis/cli/utils.py

```diff
@@ -117,8 +131,10 @@ def pretty_print_reviews(results, quiet=False):
                     )
                 if r.get("confidence") is not None:
```

Comment:
Specifying `is not None` will allow an empty string.

```suggestion
                if r.get("confidence"):
```

or

```suggestion
                if confidence := r.get("confidence"):
```